### PR TITLE
get repoID from config for events

### PR DESCRIPTION
**Pull request by neha-Gupta1**
- Get repoID from config for events for uniformity across repo
- Set a default value if repoID is nil/blank

#### Does this PR need a docs update or release note?

- [x] :no_entry: No

#### Type of change

- [ ] :bug: Bugfix

#### Issue(s)

* https://github.com/alcionai/corso/issues/3388

#### Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: src/pkg/repository/repository.go [hunk 1]
```diff
@@ -184,6 +184,7 @@ func Connect(
 	ctx context.Context,
 	acct account.Account,
 	s storage.Storage,
+	repo string,
```
Comment (vkamra) [inline src/pkg/repository/repository.go:187]: nit: This should be named `repoID`

File: src/pkg/repository/repository.go [hunk 2]
```diff
@@ -229,23 +230,21 @@ func Connect(
...
 	}
 
-	rm := &repositoryModel{}
-
 	// Do not query repo ID if metrics are disabled
 	if !opts.DisableMetrics {
-		rm, err = getRepoModel(ctx, ms)
-		if err != nil {
-			return nil, clues.New("retrieving repo info")
+		repoID := "not_found"
```
Comment (vkamra) [inline src/pkg/repository/repository.go:235]: We shouldn't initialize to `not_found` here. If there aren't any code paths where this is retrieved in this function block - we don't need that

Comment (vkamra) [reply src/pkg/repository/repository.go:235]: I would look into where we initialize `cfg` and initialize to a constant before we retrieve it from the config file

**Review by ryanfkeepers — APPROVED** [review]
some nits, approved to unblock

**Pull request merged**